### PR TITLE
SEGV in tensor_decoder with mode=pose_estimation (for out of bounds pixel).

### PR DESCRIPTION
**Pull request by tschulz**
This happened when running one of nnstreamer examples, using a different input media.

Steps to reproduce:

```
gst-launch-1.0 \
souphttpsrc location=https://storage.googleapis.com/ts-video-test/test1_vlog.mp4 \
! qtdemux ! decodebin ! videoconvert \
! videoscale ! videorate ! video/x-raw,width=640,height=360,format=RGB \
  ! tee name=nnt \
  nnt. \
    ! queue \
    ! videoscale ! videorate ! video/x-raw,width=257,height=257,format=RGB \
    ! tensor_converter \
    ! tensor_transform mode=arithmetic option=typecast:float32,add:-127.5,div:127.5 \
    ! tensor_filter framework=tensorflow-lite \
      accelerator=true:gpu,npu,cpu \
      model=posenet_mobilenet_v1_100_257x257_multi_kpt_stripped.tflite \
    ! tensor_decoder mode=pose_estimation \
      option1=640:360 \
      option2=257:257 \
      option3=labels.txt \
      option4=heatmap-offset \
  ! nnmix.sink_0 \
  nnt. \
    ! queue \
    ! nnmix.sink_1 compositor name=nnmix sink_0::zorder=2 sink_1::zorder=1 \
  ! videoconvert ! autovideosink
```
For one of the pixels is rounded to `y = 360` (equal to height) which causes a SEGV in `setpixel`

### CODE REVIEW

File: ext/nnstreamer/tensor_decoder/tensordec-pose.c [hunk 1]
```diff
@@ -513,6 +513,25 @@ typedef struct
   gfloat prob;
 } pose;
 
+/**
+ * @Brief Check if a value is within lower and upper bounds
```
Comment (anyj0527) [inline ext/nnstreamer/tensor_decoder/tensordec-pose.c:517]: "@Brief" -> "@brief" to relieve CI's complain.
You may also consider to sign-off your commit and add some commit body (other than commit head).

**Review by myungjoo — COMMENTED** [review]
Code: LGTM.

Please address CI complaints:
1. fix doxygen tag (@Brief --> @brief)
2. Fill in some commit message (it's empty). The PR message can be reused here.
3. Add signed-off-by: NAME <EMAILE> at the commit message. (Linux kernel style)

**Comment by tschulz** [comment]
Closing this pull request now, will create a clean new one. Thanks for the fast responses!

**Comment by myungjoo** [comment]
> Closing this pull request now, will create a clean new one. Thanks for the fast responses!

If you update commits with amend options and update PRs with force-push to your fork, you don't need to close it.

For example,
```
$ {edit code}
$ git commit --amend -s  # update the last commit, add signed-off
$ git push origin HEAD -f # assuming that 'origin' is your fork and the current branch is registered for the PR
```

**Pull request closed without merging**